src/main/java/net/openhft/chronicle/core/io/BackgroundResourceReleaser.java

```diff
@@ -98,7 +98,7 @@ public static void releasePendingResources() {
                     performRelease(o);
             }
             if (stopping)
-                RESOURCES.offer(POISON_PILL);
+                offerPoisonPill();
```

Comment:
this changes behaviour. We don't want to warn if we can't add - I think these 2 changes should be reverted

Reply:
Added a flag for warning